pkg/cerrl/cerrl.go

```diff
@@ -110,6 +116,16 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func isHasSuffix(path string, list []string) bool {
```

Comment:
don't need `is` before `has`. `has` already covers this
`isHasSuffix` => `hasSuffix`

Reply:
Done